configs/train_configs/sft/olmo_7b_17_remix_sft.yaml

```diff
@@ -0,0 +1,26 @@
+# Note, the exact model was trained on TPUs in a different repo
+# Example file for mixing, use exact tulu 2 mixture for replication allenai/tulu-v2-sft-mixture
+model_name_or_path: allenai/OLMo-1.7-7B-hf
+model_revision: main
+use_flash_attn: true
+tokenizer_name: allenai/OLMo-1.7-7B-hf
+use_slow_tokenizer: true
+dataset_mixer:
```

Comment:
does this config really not use any tulu data? that's surprising

Reply:
I need to ask @nouhadziri

Reply:
It doesn't, I fixed. 
Our internal wildjailbreak contains the Tulu2Mix so I thought no need to add the tulu2 data